code/game/objects/items/devices/paicard.dm

```diff
@@ -339,3 +339,7 @@
 		pai.ex_act(severity)
 	else
 		qdel(src)
+
+/obj/item/paicard/extinguish_light()
+	for(var/atom/A in src)

```

Comment:
what other atoms can be in a pAI card?

Reply:
I'm pretty sure only pAIs can be...

Reply:
if it's only the pAI, then just call `pai.extinguish_light()`

Reply:
I'm silly...how did I not notice that??